lib/src/chart/line_chart/line_chart_data.dart

```diff
@@ -1675,6 +1691,19 @@ class LineBarSpot extends FlSpot with EquatableMixin {
       ];
 }
 
+/// A [LineBarSpot] that holds information about the event that selected it
+class TouchLineBarSpot extends LineBarSpot {
```

Comment:
We document our lineChart data classes in [line_chart.md](https://github.com/imaNNeoFighT/fl_chart/blob/master/repo_files/documentations/line_chart.md) in a table. 
Please don't forget to add this class in line_chart.md (with the correct order, just like other data classes).

Reply:
I added documentation, but I am not sure about the default values. `TouchLineBarSpot` does not have default values, but it's the same for `LineBarSpot` and there it currently says `null` as default, so I did the same.